build.gradle

```diff
@@ -0,0 +1,87 @@
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'scala'
+
+    apply from: "${rootProject.projectDir}/gradle-scalatest.plugin"
+
+    ext {
```

Comment:
Generally I try to define things like this, where possible, in `gradle.properties` that way they can be more easily overwritten on the command line with `-P` if need be.

I understand that this doesn't quite work when these attributes are invoking some logic but something to take under consideration